packages/libp2p/src/connection-manager/constants.defaults.ts

```diff
@@ -1,12 +1,12 @@
 /**
  * @see https://libp2p.github.io/js-libp2p/interfaces/index._internal_.ConnectionManagerConfig.html#dialTimeout
  */
-export const DIAL_TIMEOUT = 30e3
+export const DIAL_TIMEOUT = 5e3
```

Comment:
FYI: you mentioned in PR that you were decreasing the timeout to 2s, but this is 5s